internal/services/helpers.go

```diff
@@ -134,3 +137,25 @@ func ReturnErr(w http.ResponseWriter, err string, code int) {
 	w.WriteHeader(code)
 	json.NewEncoder(w).Encode(errj)
 }
+func CreatNewTweet(newTweet tweets.CreatNewTweet, r *http.Request, userID int, tweetTo string) error {
```

Comment:
passing request around is not a good idea, if you only need a context pass a context

Reply:
done